variants/forms.py

```diff
@@ -1261,7 +1259,18 @@ class SmallVariantPrioritizerFormMixin:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.initial["prio_algorithm"] = PRIO_HIPHIVE_HUMAN
+        PRIO_ALGORITHM_CHOICES = []
+        if settings.VARFISH_ENABLE_EXOMISER_PRIORITISER:
```

Comment:
Had purposefully removed  `PRIO_ALGORITHM_CHOICES.append()` from the header to avoid duplicating if statements in header and `init()` method. The if statements are necessary, consider following scenario: If Exomiser is disabled and CADA is enabled in the settings, then the Exomiser related choices are still shown to the user and it might be confusing as the prioritization will not work when any of those options (for example: HiPhive) is selected. 

Now changing to move all lines with the append() to the header and also duplicating the if statements in header and `init()`.

Reply:
Capitalized variables defined in the header should not be changed in subsequent code other than directly after their definition (i.e. in the if statements that depend on a setting). If this involves some code duplication, this is fine. Everything else causes confusion.

Reply:
Done.

Reply:
After this change was done (moving all lines with PRIO_ALGORITHM_CHOICES.append() to the header), there were test failures for 3 of the tests in test_views.py :
Basically, PRIO_ALGORITHM_CHOICES remains empty due to the if condition in the header of the forms.py which is loaded before running the test. VARFISH_ENABLE_EXOMISER_PRIORITISER is set to True when running the test but PRIO_ALGORITHM_CHOICES remains empty and so the test fails. 

Example for one such failure:
<pre>FAIL: test_ranking_cadd_and_pheno_results (variants.tests.test_views.TestCaseLoadPrefetchedFilterView)
----------------------------------------------------------------------
Traceback (most recent call last):
  File &quot;/home/meghna/miniconda3/envs/varfish-server/lib/python3.8/unittest/mock.py&quot;, line 1325, in patched
    return func(*newargs, **newkeywargs)
  File &quot;/home/meghna/miniconda3/envs/varfish-server/lib/python3.8/site-packages/requests_mock/mocker.py&quot;, line 275, in inner
    return func(*args, **kwargs)
  File &quot;/media/meghna/Daten10/git-repo/varfish-server/variants/tests/test_views.py&quot;, line 1203, in test_ranking_cadd_and_pheno_results
    self.assertEqual(response.context[&quot;result_count&quot;], 3)
AssertionError: 2 != 3
</pre>

Reply:
@ahujameg Patch settings for a test according to what you require using `@patch` decorator, like so: https://github.com/bihealth/varfish-server/blob/db015423db29a37131d2c573867c85e95a8ab7d9/variants/tests/test_views.py#L1359

Reply:
@ahujameg Ah, you should also move the if condition and appends to the header.